### PR TITLE
Fix sunburst chart not resolved; fix data loading on dashboard

### PR DESCRIPTION
**Pull request by yizheliu-amazon**
*Issue #, if available:*

*Description of changes:*
Fix the bug of getting data request time out for sunburst chart on dashboard 
Fix sunburst chart not resolved. 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: public/pages/Dashboard/Components/AnomaliesDistribution.tsx [hunk 1]
```diff
@@ -29,7 +29,7 @@ import {
   //@ts-ignore
   EuiStat,
 } from '@elastic/eui';
-import { Chart, Partition, PartitionLayout } from '@elastic/charts';
+import { Chart, Partition, PartitionLayout } from '@elastic/charts/dist/index';
```
Comment (ohltyler) [inline public/pages/Dashboard/Components/AnomaliesDistribution.tsx:32]: wow, after I had pulled all of our latest changes I never ran into this issue again... you were getting this issue with 7.6.1 kibana oss?

Comment (yizheliu-amazon) [reply public/pages/Dashboard/Components/AnomaliesDistribution.tsx:32]: yes

Comment (yizheliu-amazon) [reply public/pages/Dashboard/Components/AnomaliesDistribution.tsx:32]: I use code from [here](https://github.com/elastic/kibana/tree/v7.6.1), but I run `yarn start --no-base-path --oss`. Will that make a difference?

Comment (ohltyler) [reply public/pages/Dashboard/Components/AnomaliesDistribution.tsx:32]: hmm no, I was doing the same. if you still encounter the issue i agree we should add this

**Pull request merged**